src/top-bar/selection/selection-context.js

```diff
@@ -0,0 +1,14 @@
+import { createContext } from 'react'
+
+const noop = () => {}
+
+const SelectionContext = createContext({
+    workflow: {},
+    setWorkflow: noop,
+    period: {},
+    setPeriod: noop,
+    orgUnit: {},
+    setOrgUnit: noop,
```

Comment:
Super minor: I think `() => throw new Error('Selection Context has not been initialized')` makes more sense than `noop` as `noop` silently fails. But I think this is really a non-issue